public/projects/markdown_preview.md

```diff
@@ -0,0 +1,30 @@
+# Markdown Preview
+
+Input some markdown source on the left and see it rendered on the right.
```

Comment:
Are you planning for this project to simply pass the input to a library, or are you planning on having the students implement a markdown->HTML rendering library themselves.

Reply:
* markdown library e.g. 
  * https://github.com/chjj/marked
  * https://github.com/showdownjs/showdown